recipes/prinseq-plus-plus/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "1.2.1" %}
+{% set sha256 = "afd115dbfb3883fcc89b3937801e6b335bcb1e95b5e06c41890713613275ac15" %}
+
+package:
+  name: prinseq-plus-plus
+  version: {{ version }}
+
+source:
+  url: https://github.com/Adrian-Cantu/PRINSEQ-plus-plus/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True # [osx]
```

Comment:
Any chance you can get it to build on MacOS, too?

Reply:
The build seems to complete fine, but it fails on the install. Unfortunately, I don't have a Mac to test this on. Is it something stupid like including a `+` in the executable name (`prinseq++`)? 

Can you take look at the failed test and make any suggestions?